### PR TITLE
federated ETL with QSR

### PR DESCRIPTION
**Pull request by jessegoodier**
Update for 1.104.1

Add configuration for using Federated ETL.

### CODE REVIEW

**Review by nik-kc — APPROVED** [review]

**Review by ameijer — DISMISSED** [review]

File: install-and-configure/install/etl-backup/query-service-replicas.md [hunk 1]
```diff
@@ -4,26 +4,26 @@
...
+There are two methods to implement QSR. For environments that have Kubecost [Federated ETL](https://docs.kubecost.com/install-and-configure/install/multi-cluster/federated-etl) enabled, the Federated ETL object-store must be used. Federated ETL is, itself, a backup mechanism.
 
-## Enabling QSR
+For all other environments, QSR will target the ETL backups. As of Kubecost v1.100+, this is enabled by default if you enable Thanos. To learn more about ETL backups, see the [ETL Backup](https://docs.kubecost.com/install-and-configure/install/etl-backup) doc.
 
-After following the doc above and enabling ETL backup, you should already have set a value for the Helm flag `.Values.kubecostModel.etlBucketConfigSecret`.
+## Enabling QSR
 
-Next, set `kubecostDeployment.queryServiceReplicas` to a non-zero value. Perform a Helm upgrade with your updated _values.yaml._
+Once Federated ETL or ETL Backups are configured, set `kubecostDeployment.queryServiceReplicas` to a non-zero value. Perform a Helm upgrade with your updated _values.yaml._
```
[the remark this answers is not in the document]
Comment (jessegoodier) [reply install-and-configure/install/etl-backup/query-service-replicas.md:25]: Totally forgot to add `queryServiceConfigSecret` thanks for reviewing this!

File: install-and-configure/install/etl-backup/query-service-replicas.md [hunk 2]
```diff
@@ -4,26 +4,26 @@
...
 
 <figure><img src="../../../.gitbook/assets/image (5).png" alt=""><figcaption></figcaption></figure>
 
 ## Prerequisites
 
-In order to make use of QSRs, you must first have enabled ETL backups. As of v1.100+ of Kubecost, this is enabled by default if you enable Thanos. To learn more about ETL backups, see the [ETL Backup](https://docs.kubecost.com/install-and-configure/install/etl-backup) doc.
+There are two methods to implement QSR. For environments that have Kubecost [Federated ETL](https://docs.kubecost.com/install-and-configure/install/multi-cluster/federated-etl) enabled, the Federated ETL object-store must be used. Federated ETL is, itself, a backup mechanism.
 
-## Enabling QSR
+For all other environments, QSR will target the ETL backups. As of Kubecost v1.100+, this is enabled by default if you enable Thanos. To learn more about ETL backups, see the [ETL Backup](https://docs.kubecost.com/install-and-configure/install/etl-backup) doc.
```
Comment (srpomeroy) [inline install-and-configure/install/etl-backup/query-service-replicas.md]: Suggest calling out ETL backup. I am reading the original sentence as QSR being enabled by default in Thanos environments.

```
As of Kubecost v1.100+, ETL Backup is enabled by default if you enable Thanos.
```

**Review by srpomeroy — DISMISSED** [review]

File: install-and-configure/install/etl-backup/query-service-replicas.md (hunk 2 above)
Comment (jessegoodier) [reply install-and-configure/install/etl-backup/query-service-replicas.md]: I'm not sure I understand the ask. This sentence is in the original and updated doc. No?

File: install-and-configure/install/etl-backup/query-service-replicas.md [hunk 3]
```diff
@@ -4,26 +4,28 @@
...
-After following the doc above and enabling ETL backup, you should already have set a value for the Helm flag `.Values.kubecostModel.etlBucketConfigSecret`.
+## Enabling QSR
 
-Next, set `kubecostDeployment.queryServiceReplicas` to a non-zero value. Perform a Helm upgrade with your updated _values.yaml._
+Once the data store is configured, set `kubecostDeployment.queryServiceReplicas` to a non-zero value and perform a Helm upgrade with your updated values.
 
 ## Usage
 
-Once ETL backups and the QSR have been enabled, the process will automatically apply to all Allocations or Assets queries. You can make these queries using either the Kubecost UI dashboards or direct API endpoints.
+Once QSR have been enabled, the new pods will automatically handle all API requests to `/model/allocation` and `/model/assets`.
```
Comment (bstuder99) [inline install-and-configure/install/etl-backup/query-service-replicas.md:31]: "Once QSR **has** been enabled..." If we are using the term as a plural, it should be QSRs.

File: install-and-configure/install/etl-backup/query-service-replicas.md (hunk 2 above)
Comment (jessegoodier) [reply install-and-configure/install/etl-backup/query-service-replicas.md]: can you review again when you have a few? I updated the section in my last commit.

**Review by nik-kc — APPROVED** [review]

**Pull request merged**